### DISCUSSION TITLE
Feature request: tag_map equivalent for dependency relations

### DISCUSSION THREAD

**Discussion by danielhers** [explosion/spaCy#2235] [New Features & Project Ideas]
The `pos` attribute is very useful for cross-lingual applications, as it has a universal set of values.
However, `dep` has different values for English (CLEAR) and German (TIGER) than other languages (UD). While I think a UD model for every language would be nice, in the meantime perhaps it would be easier to add a `dep_map` for English and German, and an attribute called something like `udep` providing the value of the mapped relation.

**Comment by honnibal** [comment]
This would make a good extension attribute, although it might be difficult to create a good mapping, since the labels won't align one-to-one.

**Comment by danielhers** [comment]
Right, the best mapping won't be 100%, but it should be pretty good nonetheless. For POS tags it's not one-to-one either, or is it?
